app/Helper/FileParserJson.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+
+namespace ArtifactCreation\Helper;
+
+use ArtifactCreation\Exception\InvalidFileException;
+use ArtifactCreation\Exception\MissingFileException;
+
+/**
+ * Parse json file to array
+ *
+ * Class Parser
+ * @package ArtifactCreation\Helper
+ */
+class FileParserJson extends FileParserAbstract
+{
+    /**
+     * Parse json file to array
+     *
+     * @return array
+     *
+     * @throws InvalidFileException
+     * @throws MissingFileException
+     */
+    public function parse()
+    {
+        $fullyQualifiedFileName = $this->file;
```

Comment:
You ca use $this->file direct instead of assign it to $fullyQualifiedFileName

Reply:
Copy pasta ;D